system-addon/test/unit/content-src/components/LinkMenu.test.jsx

```diff
@@ -31,33 +30,44 @@ describe("<LinkMenu>", () => {
     assert.ok(!lastItem.first);
 
     // middle items should have neither
-    assert.ok(!middleItem.first);
-    assert.ok(!middleItem.last);
+    for (let i = 1; i < options.length - 1; i++) {
+      assert.ok(!options[i].first && !options[i].last);
+    }
   });
-  it("should show Pin option by default (unpinned site)", () => {
-    wrapper = shallowWithIntl(<LinkMenu site={{url: ""}} source={"TOP_SITES"} dispatch={() => {}} />);
+  it("should show the correct options for default sites", () => {
+    wrapper = shallowWithIntl(<LinkMenu site={{url: "", isDefault: true}} options={["CheckBookmark"]} source={"TOP_SITES"} dispatch={() => {}} />);
     const options = wrapper.find(ContextMenu).props().options;
-    const item = options[1];
-    assert.equal("menu_action_pin", item.id);
+    assert.ok(["menu_action_pin", "menu_action_unpin"].includes(options[0].id));
+    assert.ok(options[1].type === "separator");
+    assert.ok(options[2].id === "menu_action_open_new_window");
+    assert.ok(options[3].id === "menu_action_open_private_window");
```

Comment:
maybe just add a redundant check to see that the length of our options is exactly 4? that way if any options for default get added in the future we know to change this test too